frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```
@@ -210,6 +210,10 @@ action.GitMachete.FastForwardMergeSelectedToParentAction.description=Fast-forwar
 action.GitMachete.SyncSelectedToParentByMergeAction.text=Git Machete: Checkout and Sync to Parent by Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.GitMacheteContextMenu.text=Checkout and Sync to Parent by _Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.description=Checkout this branch and merge parent into it
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.title=Merge operation warning
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message=Using merge to sync branches is discouraged, especially with stacked PRs, ignore this message and proceed if this is a hotfix branch
```

```suggestion
action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message=Using merge to sync branches is discouraged, especially with stacked PRs. Ignore this message and proceed if this is a hotfix branch
```

Actually, is it possible to add a link to the aformentioned FAQ easily to this dialog?

---

I don't think so, can't find anything about hyperlinks in the IntelliJ Platform Plugin SDK